### PR TITLE
Fixes use released version of boosters in FMP regression tests

### PR DESCRIPTION
**Pull request by piyush1594**
Fixed use a released version of all the Boosters
in place of always using master.
This will also help in regression tests breaking regularly
#1245

### CODE REVIEW

File: rt/src/test/java/io/fabric8/maven/rt/SpringbootConfigmapBoosterIT.java [hunk 1]
```diff
@@ -35,6 +34,8 @@
 
     private final String SPRING_BOOT_CONFIGMAP_BOOSTER_GIT = "https://github.com/snowdrop/spring-boot-configmap-booster.git";
 
+    private final String RELEASED_VERSION_TAG = "tags/1.5.10-5";
```
Comment (lordofthejars) [inline rt/src/test/java/io/fabric8/maven/rt/SpringbootConfigmapBoosterIT.java]: Couldn't we just put here the number part (`1.5.10-5`) and just append `tags/` inside the `setupSampleTestRepository`? So it is easier to track.

**Comment by piyush1594** [comment]
@lordofthejars Changes done. Thanks for the time.

**Comment by piyush1594** [comment]
I right now just used the last released version of the booster. I am not sure how we should do that.

**Comment by piyush1594** [comment]
@hrishin I just checked with launcher team and answer is use the latest released version of boosters in OSIO.

**Comment by piyush1594** [comment]
@hrishin @lordofthejars @rohanKanojia 

Request you to go through one more round of review. Added the fix for the concern raised by @hrishin 

The versions of Booster used by OSIO are mentioned here. https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/metadata.yaml and the respective git URL and release branch is mentioned in their respective yaml like https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/spring-boot/1.5.10-redhat/rest-http/booster.yaml

I have parsed the yaml and used the URL and git tag for regression tests. Right now, they are using the Redhat version as the info I received. This may change in the future. We just need to maintain booster.yaml URL if that changes in any case.

**Comment by piyush1594** [comment]
Using the git URL and tag from these boosters also means we are using the booster versions in sync with OSIO.

**Pull request merged**